.github/workflows/tests.yml

```diff
@@ -35,9 +35,6 @@ jobs:
       - name: Run unit tests
         run: go test -race -covermode=atomic -coverprofile=coverage.txt -coverpkg=$(go list ./... | grep -v rpmmd/test$ | tr "\n" ",") ./...
 
-      - name: Run dnfjson tests with force-dnf to make sure it's not skipped for any reason
-        run: go test -race ./internal/dnfjson/... -force-dnf
```

Comment:
I think that it would be good to keep running the unit tests with the `dnf-json` copy that lives in this repository... This will require some more thought and work.

Reply:
I wanted to comment on this but forgot, the test exists in osbuild/images https://github.com/osbuild/images/blob/main/.github/workflows/tests.yml#L40
Should we have it twice still ?

Reply:
The thing is that `images` repo has its own copy of `dnf-json`. The test there should be run there, but my point is that we should run the same test also in this repo, with the `dnf-json` copy that is in it (may be different, but should not be and running the unit test would hopefully ensure that to some extent).

Reply:
But this is a unit test of the go code which is being removed.  Running the same test would either require keeping code around that imports the code from images and then defines tests for the dependency, which is a bit weird.

Also, the python script should be removed here in favour of the `obsuild-depsolve-dnf` package that's a subpackage of osbuild.

Reply:
My whole point is that the `dnf-json` Go interface can change in `image`, together with the `dnf-json` Python script in `images` and as a result, we need to keep a form of a test that will ensure that `dnf-json` Python script in this repo keeps working with the Go interface from the `images` repo. Specifically if the the `dnf-json` script will be kept here...

Reply:
Right yeah I think we should just go ahead and remove all copies of dnf-json (the script) except for the one in osbuild and update the exec paths to use `/usr/libexec/osbuild-depsolve-dnf`.
We'll have to update the `osbuild-composer.spec` and any tests that do depsolving without osbuild (like the manifes diffs) to install that dependency.

Reply:
`dnf-json` is now removed from images as well.  Resolving this.